README.md

```diff
@@ -287,6 +275,50 @@ spec:
 ...
 ```
 
+### Installing on a Custom Namespace with RBAC enabled
+
+Create the namespace.
+
+```
+$ kubectl create namespace pravega-io
+```
+
+Update the namespace configured in the `deploy/role_binding.yaml` file.
+
+```
+$ sed -i -e 's/namespace: default/namespace: pravega-io/g' deploy/role_binding.yaml
+```
+
+Apply the changes.
+
+```
+$ kubectl -n pravega-io apply -f deploy
+```
+
+Note that the Pravega operator only monitors the `PravegaCluster` resources which are created in the namespace where it was deployed, so if you want to create a cluster you have to specify the same namespace as the operator.
```

Comment:
This comment is not making very clear how one achieves it. It might be better to be more concrete and say something like: In the command below to create a cluster, we use `-n pravega-io` to specify that the namespace is `pravega-io`,  the same as the operator one.

Reply:
I rephrased the entire paragraph to:

> Note that the Pravega operator only monitors the `PravegaCluster` resources which are created in the same namespace, `pravega-io` in this example. Therefore, before creating a `PravegaCluster` resource, make sure an operator exists in that namespace.